buffalo/cmd/filetests/version-dep.json

```diff
@@ -0,0 +1,6 @@
+[{
+  "path": "output.txt",
+  "contains": [
+    "v0.11.0 [development]"
```

Comment:
This will fail every time we have a new release. Got anything else you can use here instead?